service/index/metrics.go

```diff
@@ -0,0 +1,183 @@
+// Copyright 2021 Optakt Labs OÜ
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+package index
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+
+	"github.com/onflow/flow-go/ledger"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+const (
+	labelBlockID       = "blockid"
+	labelPath          = "registerpath"
+	labelCollectionID  = "collectionid"
+	labelTransactionID = "transactionid"
+	labelEventID       = "eventid"
+	labelSealID        = "sealid"
```

Comment:
For logging, and by extension metrics, it's customary to use fields without the `id` suffix, as you can't log the whole thing anyway. So the ID is like a representation for the whole thing. Same as the path is for the register.

Therefore:

- `block`
- `register`
- `collection`
- `transaction`
- `event`
- `seal`